frontend/src/views/LevelView.vue

```diff
@@ -0,0 +1,183 @@
+<template>
+  <div class="PageWrapper">
+    <div class="Container">
+      <img src="@/assets/images/logo-and-lettering.svg" alt="Saci Logo" />
+      <h1 style="padding-bottom: 30px">{{ newLevelTitle }}</h1>
+      <h2>Basic Information</h2>
+      <div class="NameContainer">
+        <h5 style="padding-bottom: 5px">Name</h5>
+        <ElInput style="width: 800px; height: 56px" placeholder="Level Name" v-model="levelName" />
+      </div>
+      <div class="FlexContainer">
+        <div class="MaxAndMinInputs">
+          <h5 style="padding-bottom: 5px">Coeficient Range</h5>
+          <span>
+            <ElInput
+              style="width: 110px; height: 56px"
+              placeholder="Min Range"
+              v-model="levelMinRange"
+            ></ElInput>
+            <h4>to</h4>
+            <ElInput
+              style="width: 110px; height: 56px"
+              placeholder="Max Range"
+              v-model="levelMaxRange"
+            ></ElInput>
+          </span>
+        </div>
+        <div class="LinkSide">
+          <h5 style="padding-bottom: 5px">Link to Knowledge Path</h5>
+          <ElInput
+            style="width: 417px; height: 56px"
+            placeholder="Link"
+            v-model="levelLink"
+          ></ElInput>
+        </div>
+      </div>
+      <div class="SaveButton">
+        <ElButton
+          type="primary"
+          style="width: 215px; height: 32; border-radius: 4px"
+          @click="CreateLevel()"
+        >
+          Save
+        </ElButton>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import type { Role } from '@/domain/Role'
+import type { Level } from '@/domain/Level'
+import { useRolesStore } from '@/stores/roles'
+import { ref, onMounted, computed } from 'vue'
+import { ElInput, ElButton } from 'element-plus'
+import { ErrorMessage, SuccessMessage } from '@/services/messages'
+import { useRouter } from 'vue-router'
+import { HttpServerError } from '@/services/http'
+
+const roleStore = useRolesStore()
+const roleId = ref<string>('1')
+const role = ref<Role>()
+const levelName = ref<string>('')
+const levelMinRange = ref<string>('')
+const levelMaxRange = ref<string>('')
+const levelLink = ref<string>('')
+const newLevel = ref<Level>({ name: '', minCoeficient: 0, maxCoeficient: 1, link: '' })
+const router = useRouter()
+const title = ref<string>('New Level')
```

Comment:
You don't need this variable. You can put the value directly in the computed field